controller/pokemon_test.go

```diff
@@ -0,0 +1,231 @@
+package controller
```

Comment:
If you're using vscode, and have the go plugin installed along with the go tools, you can generate test templates automatically per function or for a whole file by doing Cmd+Shift+P and typing something like `generate test`, it should display a list of options and one of them should look like `Generate Test for Function / File`

Reply:
👌